src/reaper_osara.cpp

```diff
@@ -3809,17 +3809,47 @@ void cmdDeleteTimeSig(Command* command) {
 		outputMessage(translate("time signature deleted"));
 }
 
-void cmdRemoveStretch(Command* command) {
-	MediaItem* item = GetSelectedMediaItem(0, 0);
-	if (!item)
-		return;
-	MediaItem_Take* take = GetActiveTake(item);
-	if (!take)
+void cmdhAddOrRemoveStretch(int command) {
+	const int itemCount = CountSelectedMediaItems(nullptr);
+	if (itemCount == 0) {
+		outputMessage(translate("no selected items"));
 		return;
-	int count = GetTakeNumStretchMarkers(take);
-	Main_OnCommand(41859, 0); // Item: remove stretch marker at current position
-	if (GetTakeNumStretchMarkers(take) != count)
-		outputMessage(translate("stretch marker deleted"));
+	}
+	int oldCount = 0;
+	for (int i = 0; i < itemCount; ++i) {
+		MediaItem* item = GetSelectedMediaItem(nullptr, i);
```

Comment:
Just to clarify, these commands definitely insert stretch markers across all selected items? I guess that makes sense and we just never supported this properly, but I just wanted to check.